plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/MigrateKVMAsync.java

```diff
@@ -51,19 +76,19 @@ public MigrateKVMAsync(final LibvirtComputingResource libvirtComputingResource,
 
     @Override
     public Domain call() throws LibvirtException {
-        long flags = 1 << 0;
+        long flags = VIR_MIGRATE_LIVE;
 
         // set compression flag for migration, if libvirt version supports it
         if (dconn.getLibVirVersion() >= 1000003) {
-            flags |= 1 << 11;
```

Comment:
Do you also want to make the version a final Integer?

Reply:
Agree, will do that. Thanks!